server/routes/bookedtoday/arrivals/confirmArrivalController.test.ts

```diff
@@ -42,4 +44,47 @@ describe('GET /confirmArrival', () => {
         expect(res.header['set-cookie'][0]).not.toContain('movementReasonCode')
       })
   })
+
+  it('should display correct page heading when there is NOT an existing prisoner record', () => {
```

Comment:
Difficult to test page header content in integration tests because header is h1 and it is dynamic. Easier to test as unit test